### PR TITLE
add filters row to chart image in MO

### LINKED ISSUE #313: [MO] Labels on page of chart exports showing what filters the user has selected
**Is your feature request related to a problem? Please describe.**
User experience enhancement when exporting a table.

**Describe the solution you'd like**
Add labels on the page of chart exports showing what filters the user has selected, e.g., most severe violation = technical, supervision type = parole, time period = 1 year. 

**Describe alternatives you've considered**
N/A

### PR DESCRIPTION
**Pull request by DmitryJakubovsky**
## Description of the change

Added new line with filters to exported chart image.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #313

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/assets/scripts/utils/downloads.js [hunk 1]
```diff
@@ -62,6 +62,14 @@ function downloadCanvasImage(canvas, filename, chartTitle, shouldZipDownload) {
   destinationCtx.textAlign = 'center';
   destinationCtx.font = '30px Helvetica Neue';
   destinationCtx.fillText(chartTitle, canvas.width / 2, topPadding / 2);
+
+  if (toggleStates) {
+    destinationCtx.fillStyle = '#B8B8B8';
+    destinationCtx.textAlign = 'center';
+    destinationCtx.font = '13px Helvetica Neue';
+    destinationCtx.fillText(`Applied filters: ${formatFilters(toggleStates)}`, canvas.width / 2, topPadding - 20);
```
Comment (jessex) [inline src/assets/scripts/utils/downloads.js:70]: We also want to include the two filter states that come from interacting with the revocation grid, e.g. number of violations or notices of citation and most severe violation, as labeled in the methodology file via the `getViolation` function. We could do it either all on one line or on a second line, whichever makes more visual sense.

(I know that wasn't _explicitly_ stated in #313, but it was listed in the example. We'll make sure issues leave no ambiguity in the future.)

File: src/assets/scripts/utils/downloads.js [hunk 2]
```diff
@@ -62,6 +62,14 @@ function downloadCanvasImage(canvas, filename, chartTitle, shouldZipDownload) {
   destinationCtx.textAlign = 'center';
   destinationCtx.font = '30px Helvetica Neue';
   destinationCtx.fillText(chartTitle, canvas.width / 2, topPadding / 2);
+
+  if (toggleStates) {
+    destinationCtx.fillStyle = '#B8B8B8';
+    destinationCtx.textAlign = 'center';
+    destinationCtx.font = '13px Helvetica Neue';
```
Comment (jessex) [inline src/assets/scripts/utils/downloads.js:69]: Can we make this slightly larger? I think many users, especially older ones, will need to lean into the screen to see the text at this size. It should still definitely be smaller than the title text, though.

Comment (DmitryJakubovsky) [reply src/assets/scripts/utils/downloads.js]: Increased by 3px.

File: src/assets/scripts/utils/downloads.js (hunk 1 above)
Comment (DmitryJakubovsky) [reply src/assets/scripts/utils/downloads.js]: Done.

File: src/assets/scripts/utils/downloads.js [hunk 3]
```diff
@@ -85,65 +86,79 @@ function downloadCanvasImage(canvas, filename, chartTitle, toggleStates, shouldZ
...
+  let text = `Chart: ${chartTitle}\r\n`;
+  text += `Dates: ${timeWindowDescription}\r\n`;
+  text += `Applied filters:\r\n`;
+  text += `- ${filters}\r\n`;
+
+  if (violation) {
+    text += `- ${violation}\r\n\r\n`;
+  }
+
+  text += `Export Date: ${exportDate}\r\n`;
```
Comment (jessex) [inline src/assets/scripts/utils/downloads.js:147]: In this change we accidentally lose a line of whitespace after the Export Date line and before the additional info copy. We should add one more newline to the end of this one.

File: src/assets/scripts/utils/downloads.js [hunk 4]
```diff
@@ -85,65 +86,79 @@ function downloadCanvasImage(canvas, filename, chartTitle, toggleStates, shouldZ
...
+    toggleStates.reportedViolations !== undefined ||
+    toggleStates.violationType !== undefined ||
+    toggleStates.reportedViolations !== "" ||
+    toggleStates.violationType !== ""
+  ) {
     if (toggleStates.reportedViolations !== undefined && toggleStates.reportedViolations !== "") {
       str += toggleStates.reportedViolations + " violations or notices of citation, ";
     }
     if (toggleStates.violationType !== undefined && toggleStates.violationType !== "") {
       str += "Most severe: " + toHumanReadable(toTitleCase(toggleStates.violationType.toLowerCase()));
```
Comment (jessex) [inline src/assets/scripts/utils/downloads.js:125]: Slight functionality change over what already exists, but let's update this string to "Most severe violation". That will read a bit easier.

Comment (DmitryJakubovsky) [reply src/assets/scripts/utils/downloads.js]: Added.

File: src/assets/scripts/utils/downloads.js (hunk 3 above)
Comment (DmitryJakubovsky) [reply src/assets/scripts/utils/downloads.js:148]: Fixed.

File: src/assets/scripts/utils/downloads.js (hunk 4 above)
Comment (jessex) [reply src/assets/scripts/utils/downloads.js]: It looks like this got lost in your "Resolved conflicts" commit, unfortunately, and it reverted back to just "Most severe." Would you mind quickly making that addition again? Then we can merge this.

Comment (DmitryJakubovsky) [reply src/assets/scripts/utils/downloads.js]: Sorry for my negligence. Done.

**Pull request merged**